pkgs/by-name/ld/ld-audit-search-mod/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  storeDir ? builtins.storeDir,
+  spdlog,
+  yaml-cpp,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ld-audit-search-mod";
+  version = "0-unstable-2025-06-19";
+
+  src = fetchFromGitHub {
+    repo = finalAttrs.pname;
```

Comment:
I don't think it makes sense to reuse `finalAttrs.pname` here: if this gets modified with overrideAttrs, you'd unintentionally select a different repo here (that probably doesn't exist).